### PR TITLE
refactor: transactions truncated name tooltip

### PR DESCRIPTION
**Pull request by alexbarnsley**
## Summary

https://app.clickup.com/t/862kcujfc

In reality, the "To" address doesn't need a tooltip, but I added it for consistency.

## Checklist

-   [ ] I checked my UI changes against the design and there are no notable differences
-   [ ] I checked my UI changes for any responsiveness issues
-   [ ] I checked my UI changes in light AND dark mode
-   [ ] I checked my (code) changes for obvious issues, debug statements and commented code
-   [ ] I opened a corresponding card on Clickup for any remaining TODOs in my code
-   [ ] I added a short description on how to test this PR _(if necessary)_
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [ ] Ready to be merged

### CODE REVIEW

**Review by samharperpittam — COMMENTED** [review]
I'm a bit unsure on the implementation for this. Currently it's displaying a tooltip over names that aren't truncated which feels a little redundant to me

**Comment by ItsANameToo** [comment]
@alexbarnsley I added a component to dynamically show the tooltip, please have a look as it may need some cleanup/second pair of eyes

**Review by samharperpittam — APPROVED** [review]

File: resources/views/components/dynamic-tooltip.blade.php [hunk 1]
```diff
@@ -0,0 +1,41 @@
+@props([
+    'tooltip',
+])
+
+<div class="min-w-0"
```
Comment (alexbarnsley) [inline resources/views/components/dynamic-tooltip.blade.php:5]:
```suggestion
<div 
    class="min-w-0"
```

**Comment by alexbarnsley** [comment]
I approve @ItsANameToo

**Pull request merged**